src/data/Data.js

```diff
@@ -735,15 +736,19 @@ export const WorkspaceData = _.flow(
         .Apps.listAppsV2(workspaceId)
         .then((apps) => {
           const foundApp = resolveWdsApp(apps);
-          if (foundApp?.status === 'RUNNING') {
+          if (foundApp?.status === appStatuses.running.status) {
             const url = foundApp.proxyUrls.wds;
             setWdsProxyUrl({ status: 'Ready', state: url });
             return url;
           }
-          if (foundApp?.status === 'ERROR') {
+          if (foundApp?.status === appStatuses.error.status) {
             setWdsProxyUrl({ status: 'Error', state: 'WDS app is in ERROR state' });
           }
           return '';
+        })
+        .catch((error) => {
+          setWdsTypes({ status: 'Error', state: 'Error resolving WDS app' });
```

Comment:
```suggestion
          setWdsProxyUrl({ status: 'Error', state: 'Error resolving WDS app' });
```

Since this is `loadWdsUrl`, should this be setting the proxy URL state instead of the types state?

Reply:
Yes, totally...this was just a mistake.  Oops!